src/components/utils/src/qdb_wrapper/qdbserver.sh

```diff
@@ -1,6 +1,30 @@
 # This script star QDB server for SDL
 # Need superuser to start qdb
-
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/qnx650/target/qnx6/x86/usr/lib
-export LD_LIBRARY_PATH
-/usr/sbin/qdb -c policy.ini
+POLICY_PATH=$PWD/$(echo "$a" | sed -n 's/^AppStorageFolder = //p' $PWD/smartDeviceLink.ini)
```

Comment:
@SNiukalov where variable `a` defined?

Reply:
Thanks, this rudiment, fixed in https://github.com/Ford-Luxoft/sdl_core/commit/c448683f13d6186babde36bbc3850e91f97bf565